templates/components/hero/hero.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file
+ * Hero Component
+ *
+ * A big graphic for use as a focal point or a large video.
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * - hero_media: iframe or HTML5 video players
+ * - hero_image: A full HTML entity for an image. Can be img or picture.
+ * - hero_super_headline: Text heading
+ * - hero_headline: Text heading
+ * - hero_body: An open variable for anything. Generally this would be for
+ *     WYSIWYG editor contents.
+ * - hero_cta_link: The href value for the cta field link
+ * - hero_cta_attributes: Any html attributes on the cta link that need to be
+ *     added. For example: _target.
+ * - hero_cta_label: The clickable text for the CTA link
+ * - hero_button_link: The href property of the button component.
+ * - hero_button_label: The text label of the button component.
+ */
+#}
+
+{#
+  Sanitize the Drupal
+#}
+{%- if hero_super_headline|render|striptags is empty -%}
```

Comment:
according to https://www.lullabot.com/articles/level-your-twiggery you always want to keep `<drupal-render-placeholder>` tags when doing this. Its also mentioned [here](https://www.drupal.org/project/drupal/issues/2672656#comment-13091978)

Reply:
💡 Yes, thank you.